frontend/src/general/dashboard/__tests__/DashboardCard.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { describe, expect, it } from "@jest/globals";
+import { Typography, Paper } from "@material-ui/core";
+import DashboardCard from "../DashboardCard";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe.only("<DashboardCard/> Component", () => {
+  let wrapper;
+  const dummyTitle = "Title";
+  const dummyBody = "Body";
+
+  describe(`render() with props title:${dummyTitle} children:${dummyBody}`, () => {
+    wrapper = Enzyme.shallow(
+      <DashboardCard title={dummyTitle}>{dummyBody}</DashboardCard>
+    );
+
+    describe("Contains 1 <Paper/> component", () => {
+      expect(wrapper.find(Paper)).toHaveLength(1);
+      it(`That has text "${dummyBody}"`, () => {
+        const bodyBox = wrapper.find(Paper);
+        expect(bodyBox.text());
+      });
+    });
+    describe("Contains 1 <Typography/> component", () => {
+      expect(wrapper.find(Typography)).toHaveLength(1);
+      it(`That has title "${dummyTitle}"`, () => {
+        const bodyBox = wrapper.find(Paper);
+        expect(bodyBox.text());
+      });
+    });
```

Comment:
Stylistic suggestion for tests:
- Place the creation of the wrapper in a `beforeEach` block, to ensure that it gets re-created before each test and thus ensuring tests do not leak
- expect should only be done inside an "it" block, otherwise you're asserting before the test even starts

Reply:
adapted in 

- fe0c12a
- d8df198
- 3633b61
- 6c24eee3
- a3098d97
- 9a8443c4